milli/src/error.rs

```diff
@@ -216,6 +218,7 @@ impl fmt::Display for UserError {
                 )
             }
             Self::InvalidFilter(error) => error.fmt(f),
+            Self::InvalidAscDescSyntax { name } => write!(f, "{}", name),
```

Comment:
Not the expected error message.

Reply:
What message do you want to return?
To me we are not supposed to return anything here so the caller can then do a `to_string` on the error and wrap it into either a InvalidSort or a InvalideCriterion

Reply:
Hum, as it is an error I would like to have a message in front of the invalid syntax, something like "invalid asc/desc syntax for {}". The receiver of this error variant can just pattern match and retrieve the `name` field and do what he want.

Reply:
done